Alignment/APEEstimation/test/cfgTemplate/apeEstimator_cfg.py

```diff
@@ -168,49 +172,31 @@
 if isParticleGun:
     process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_design', '')
 elif isMc:
-    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_design', '')
+    #~ process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_design', '')
+    #~ process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2017_design', '')
+    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2017_realistic', '')
     
-    ##### To be used when running on Phys14MC with a CMSSW version > 72X
-    # process.GlobalTag.toGet = cms.VPSet(
-		# cms.PSet(
-			# record = cms.string("BeamSpotObjectsRcd"),
-			# tag = cms.string("Realistic8TeVCollisions_START50_V13_v1_mc"),
-			# connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS')
-		# )
-	# )
-
 
 elif isData:
-    #~ process.GlobalTag.globaltag = 'GR_P_V56'
     process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')
 
 
 
 
 ## Alignment and APE
 ##
-import CalibTracker.Configuration.Common.PoolDBESSource_cfi
 ## Choose Alignment (w/o touching APE)
 if options.alignRcd=='design':
-  process.myTrackerAlignment = CalibTracker.Configuration.Common.PoolDBESSource_cfi.poolDBESSource.clone(
-    connect = 'frontier://FrontierProd/CMS_CONDITIONS',
-    toGet = cms.VPSet(
-      cms.PSet(
-        record = cms.string('TrackerAlignmentRcd'),
-        tag = cms.string('TrackerAlignment_Ideal62X_mc')
-      )
-    )
-  )
-  process.es_prefer_trackerAlignment = cms.ESPrefer("PoolDBESSource","myTrackerAlignment")
+	pass
 
   
 elif options.alignRcd == 'misalTest':
   process.myTrackerAlignment = CalibTracker.Configuration.Common.PoolDBESSource_cfi.poolDBESSource.clone(
```

Comment:
please replace this snippet with sth. like this:
```
process.GlobalTag.toGet.append(
    cms.PSet(
        record = cms.string("TrackerAlignmentRcd"),
	tag = cms.string("TrackerAlignment_Phase1Realignment_CRUZET_2M"),
	connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS')
    )
)
```
If you want this GT-customization in a tracked fashion you could use something like [this](https://github.com/cms-sw/cmssw/blob/master/Alignment/OfflineValidation/test/GeometryCentering_cfg.py#L14-L26).